pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1879,9 +1902,11 @@ type AlertmanagerEndpoints struct {
 	Sigv4 *Sigv4 `json:"sigv4,omitempty"`
 
 	// Version of the Alertmanager API that Prometheus uses to send alerts.
-	// It can be "v1" or "v2".
+	// It can be "v1"/"V1" or "v2"/"V2".
```

Comment:
(nit) we can only document the values following the Kube API conventions (e.g. upper-case).
```suggestion
	// It can be `V1` or `V2`.
```

Reply:
Done.